src/summary/history/history-section.component.tsx

```diff
@@ -2,38 +2,20 @@ import React from "react";
 import { match } from "react-router";
 import SummarySectionCards from "../cards/summary-section-cards.component";
 import AllergyCard from "./allergy-card.component";
-import { getCurrentPatient } from "@openmrs/esm-api";
 import ConditionsCard from "./conditions-card.component";
 import NotesCard from "./notes-card.component";
 
 export default function HistorySection(props: HistorySectionProps) {
-  const [currentPatient, setCurrentPatient] = React.useState(null);
-
-  React.useEffect(() => {
-    const subscription = getCurrentPatient().subscribe(patient =>
-      setCurrentPatient(patient)
-    );
-
-    return () => subscription.unsubscribe();
-  });
-
   return (
-    <>
-      {currentPatient && (
```

Comment:
The child components can now handle being rendered before the current patient is loaded